download-chromedriver.js

```diff
@@ -27,12 +25,22 @@ function processDownload (err, zipPath) {
   })
 }
```

Comment:
FYI, [`extract-zip` 2.0.0](https://github.com/maxogden/extract-zip/releases/tag/v2.0.0) has been promisified as well. (Although that necessitates requiring Node 10, so maybe use `util.promisify`...)